lib/controllers/src/ActuatorController.cpp

```diff
@@ -217,4 +219,133 @@ bool ActuatorController::isPastMinAngle() {
 
 bool ActuatorController::isPastMaxAngle() {
   return (getAngle_Degrees() > m_actuatorConfig.maxAngle_Degrees || isLimSwitchMaxTriggered());
-}
\ No newline at end of file
+}
+
+// Update m_actuatorConfig upon reception of new PID values
+void ActuatorController::updateActuatorConfigP(const float newP, const bool isVelocityPID) {
+  isVelocityPID ? m_actuatorConfig.velocityPID.P : m_actuatorConfig.positionPID.P = newP;
+}
+
+void ActuatorController::updateActuatorConfigI(const float newI, const bool isVelocityPID) {
+  isVelocityPID ? m_actuatorConfig.velocityPID.I : m_actuatorConfig.positionPID.I = newI;
```

Comment:
do you really need a ternary if statement here? Could you just use `if (!isVelocityPID) m_actuatorConfig.positionPID.I = newI; `